modules/network-observability-enriched-flows-kafka.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// network_observability/understanding-network-observability-operator.adoc
+
+:_content-type: Procedure
+[id="network-observability-enriched-flows-kafka_{context}"]
+= Export enriched network flow data
+
+You can send network flows to Kafka, so that they can be consumed by any processor or storage that supports Kafka input, such as Splunk, Elasticsearch, or Fluentd,
+
+.Prerequisites
+* Installed Kafka
+
+.Procedure
+
+. Edit the `FlowCollector` to configure `spec.exporters` as follows:
++
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  exporters:
+  - type: KAFKA
+      kafka:
+        address: "kafka-cluster-kafka-bootstrap.netobserv"
+        topic: netobserv-flows-export   <1>
+----
+<1> NetObserv exports all flows to the configured Kafka topic.
```

Comment:
It's confusing to have two "1." which looks like it's part of an outline and a "\<1>" in the YAML.  Can it look something like this?
[https://docs.openshift.com/container-platform/4.12/logging/cluster-logging-loki.html#logging-loki-retention_cluster-logging-loki](https://docs.openshift.com/container-platform/4.12/logging/cluster-logging-loki.html#logging-loki-retention_cluster-logging-loki)  
  
This uses a black circle with a white font to explain items in the manifest.  It also has better colors.

Reply:
These "<1>" render as a black circle with white font. In my preview link, you can see this:
![image](https://user-images.githubusercontent.com/77501157/229617117-98e5a25e-8b72-4195-a82e-5bba02348b3c.png)